languages/csharp/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,42 @@
+In C#, data structures that can hold zero or more elements are known as _collections_. An array is a collection with a fixed size and which elements must all be of the same type. Elements in an array can be assigned to and retrieved by index, with the first element's index being zero:
+
+```csharp
+// Declare array with explicit size (size is 2)
+int[] twoIntegers = new int[2];
+
+// Assign first and second element by index
+twoIntegers[0] = 7;
+twoIntegers[1] = 8;
+
+// Retrieve the second element by index
+twoIntegers[1] == 8; // => true
+```
+
+Arrays can also be defined using a shortcut notation that allows you to both create the array and set its value. As the compiler can now tell how many elements the array will have, the length can be omitted:
+
+```csharp
+// Assign and initialize (size is 3)
+double[] threeDoublesV1 = new double[] { 1.2, 4.45, 9.998 };
```

Comment:
I suggest using integers, not floating points in these example. It makes it easier to parse the `.` vs `,`.

Reply:
Yeah, I wanted to show use a different type, but integers are clearer.